packages/rfw/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: rfw
 description: "Remote Flutter widgets: a library for rendering declarative widget description files at runtime."
 repository: https://github.com/flutter/packages/tree/main/packages/rfw
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+rfw%22
-version: 1.0.5
+version: 1.0.6
```

Comment:
This doesn't need a version bump since it's test-only. CI is a false positive since it's an unusual package structure.

You can refer the version change and add the override label instead.

Reply:
Does it require a next?

Reply:
... which will need https://github.com/flutter/packages/pull/2494 to land. Oops.